src/app/services/articles-service/index.ts

```diff
@@ -0,0 +1,46 @@
+import dayjs from 'dayjs';
+// eslint-disable-next-line
+// @ts-ignore
```

Comment:
ts ignore?

Reply:
It's a workaround of `react-native-dotenv` incompatibility with Typescript.
I've provided an explanation but in `config.ts` file below.